Description: Fix timing-quotient test
 This switch is turned on by default in Singular >= 4.1.2, so we can remove
 this patch once Singular has been updated in Debian.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1316
Last-Update: 2020-06-22

--- a/M2/Macaulay2/e/x-factor.cpp
+++ b/M2/Macaulay2/e/x-factor.cpp
@@ -102,6 +102,7 @@
 {
   oldcharac = getCharacteristic();
   oldRatlState = isOn(SW_RATIONAL);
+  On(SW_USE_EZGCD_P);
   switch (mode)
     {
       case modeZZ:
